volatility3/framework/symbols/windows/extensions/__init__.py

```diff
@@ -574,13 +574,9 @@ def get_peb(self) -> interfaces.objects.ObjectInterface:
         proc_layer = self._context.layers[proc_layer_name]
         if not proc_layer.is_valid(self.Peb):
             raise exceptions.InvalidAddressException(proc_layer_name, self.Peb,
-                                                     f"Invalid address at {self.Peb:0x}")
+                                                     f"Invalid Peb address at {self.Peb:0x}")
 
-        sym_table = self.vol.type_name.split(constants.BANG)[0]
-        peb = self._context.object(f"{sym_table}{constants.BANG}_PEB",
-                                   layer_name = proc_layer_name,
-                                   offset = self.Peb)
-        return peb
+        return self.at_layer(proc_layer_name).Peb
```

Comment:
As the only example to use the new method, this is not worth the changes.  I also think it's a pretty uncommon situation to want an object instantiated at the same offset in a mapping that potentially maps that offset to a totally different place.

Reply:
removed the function